### PR TITLE
Issue 2018/04

### PR DESCRIPTION
**Pull request by garbas**
I'm creating this so there's one place to submit topics for the next issue.
Editor is @garbas

- Please comment with a topic

- Date: 2018-03-12

- [Click to Preview](http://weekly.nixos.org/preview)

### CODE REVIEW

**Comment by grahamc** [comment]
_hem_ https://github.com/NixOS/rfcs/pull/25#event-1501334301

**Comment by garbas** [comment]
Let's make Impala the best release so far!

[18.03 Zero Hydra Failures (generic + x86_64-linux)](https://github.com/NixOS/nixpkgs/issues/36453)
[18.03 Zero Hydra Failures - aarch64 edition](https://github.com/NixOS/nixpkgs/issues/32326)
[18.03 Zero Hydra Failures - Darwin edition](https://github.com/NixOS/nixpkgs/issues/36454)

**Comment by badmutex** [comment]
~London Meetup [18.03 install party](https://www.meetup.com/NixOS-London/events/248378212/)~

---

edit: i see that this is already in the issue :)

**Comment by badmutex** [comment]
[Support fetching docker images from V2 registries](https://github.com/NixOS/nixpkgs/pull/32248) merged

**Comment by zimbatm** [comment]
NixCon 2018 in September: https://twitter.com/nixcon2018/status/972475347454382081
We're looking for help and sponsors!

**Comment by zimbatm** [comment]
https://github.com/siers/nix-gitignore - filterSource from .gitignore

**Comment by AndersonTorres** [comment]
What about something explaining "the life after Nix 2.0"?

**Comment by garbas** [comment]
@AndersonTorres the newsletter is only a collection of links and titles. if somebody writes it, I don't see a reason why we wouldn't include it.

That said, i'm one day late. But I should have it released by the end of the day.

File: content/2018/04-next.rst [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+.. 
+.. .. _`with link`: http://example.com
+
+
+- `Patreon to support the NixOS OfBorg infrastructure`_
+
+  This supports `@grhmc`_\'s work on the OfBorg tooling.
+
+.. _`Patreon to support the NixOS OfBorg infrastructure`: https://www.patreon.com/ofborg/overview
+.. _`@grhmc`: https://twitter.com/grhmc/status/967531653387169792
```
Comment (Mic92) [inline content/2018/04-next.rst]: @grahamc delete his tweets regularly, also this one.

**Comment by zimbatm** [comment]
http://nmattia.com/posts/2018-03-21-nix-reproducible-setup-linux-macos.html

**Comment by CMCDragonkai** [comment]
https://matrix.ai/2018/03/24/developing-with-nix/

**Comment by domenkozar** [comment]
I'm rebooting this project as I think it's very valuable to the community. Let's start simple, I'd like to first familiarize with the process so first weekly goes out on Tuesday ~10pm UTC+2.

**Comment by grahamc** [comment]
https://grahamc.com/blog/cache-nixos-org-now-more-local

**Comment by domenkozar** [comment]
Ah one thing: @grahamc can you provide 1-2 sentences that would highlight this blog post? I'm aiming to steal Features section from https://haskellweekly.news/issues/116.html

Same goes for the rest of the links :)

**Comment by domenkozar** [comment]
Change of plans, weekly will go out tomorrow due to some of my personal stuff and I need to sync up with Garbas on the process.

**Comment by grahamc** [comment]
I won’t Be able to do that today. Sorry.

**Review by domenkozar — APPROVED** [review]

**Comment by domenkozar** [comment]
@grahamc no worries, we'll improve in next editions :)

**Pull request merged**